File: Module/Feeds/src/main/java/com/module/feeds/room/msg/process/FeedsMsgProcess.java
package com.module.feeds.room.msg.process;

import com.common.log.MyLog;
import com.module.msg.IPushMsgProcess;

public class FeedsMsgProcess implements IPushMsgProcess {
    public final static String TAG = "ChatRoomMsgProcess";

    @Override
    public void process(int messageType, byte[] data) {
        MyLog.d(TAG, "process" + " messageType=" + messageType + " data=" + data);
        switch (messageType) {
        }

    }

    @Override
    public int[] acceptType() {
        return new int[]{
        };
    }

    // 处理房间消息
    private void processRoomMsg(byte[] data) {

    }
}

File: Module/Feeds/src/main/java/com/module/feeds/room/msg/process/FeedsMsgProcess.kt
package com.module.feeds.room.msg.process

import com.common.log.MyLog
import com.module.msg.IPushMsgProcess

class FeedsMsgProcess : IPushMsgProcess {
    val TAG = "FeedsMsgProcess"

    override fun process(messageType: Int, data: ByteArray) {
        MyLog.d(TAG, "process messageType=$messageType data=$data")
        when (messageType) {

        }

    }

    override fun acceptType(): IntArray {
        return intArrayOf()
    }

    // 处理房间消息
    private fun processRoomMsg(data: ByteArray) {

    }

}
